release-content/release-notes/atmosphere_environment_map.md

```diff
@@ -1,16 +1,16 @@
 ---
-title: Generated environment map for procedural atmosphere
+title: Generate environment maps from a procedural atmosphere
```

Comment:
```suggestion
title: Runtime generated environment maps
```
This is not specific to atmospheres, and the point is that you can now generate them on the fly. This means you can do IBL from in-scene dynamic reflection probes, or user-uploaded environment maps, not just atmosphere!

Im not sure what the best way to communicate this is, but i'd like to highlight this is not atmosphere specific.

Reply:
Actually oops, is there a different release note for that? Maybe we want to unify them.

Reply:
Yeah, we can merge those 🤔